app.js

```diff
@@ -33,7 +33,14 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 if (env === 'production' || env === 'test') {
     var RedisStore = require('connect-redis')(session);
-    var redisClient = redis.createClient();
+    var redisClient = redis.createClient(
```

Comment:
should anyone even use it?

Reply:
not in local setup. I have added a redis cache only in prod